### DISCUSSION TITLE
Fysetc SD-Wifi Pro File Transfer Issues

### DISCUSSION THREAD

**Discussion by y2kdread** [luc-github/ESP3D#983] [General]
@luc-github  - 

I didn't want to open a know bug for this, but wanted to let you know that I purchased a second SD-Wifi Pro and still had the same issue. Would it be possible to mail you one of the two SD cards in question so you can replicate the issue on your end?

I don't need it sent back, just want to try and help others who may run into the same issue.

Thanks!

**Comment by luc-github** [comment]
I do not think your issue is only hardware - I have 2 here and same behavior ( I always order 2 samples to sort hardware issues) - I cannot duplicate 
Your issue is may be:
 * a signal issue as this WiFi SD has no antenna, on my side the AP is at 40cm so signal is excellent
 * a client issue - as mentioned the esp3d can only handle one connection at a time so multiple connection access may be failed resulting lost data if client do not wait for a retry 
 * context usage, ESP3D is made for basic file transfer issue not large amount of files at once, so handling 700+ at once may lead to some situations as it was not really tested until today
 * hardware could be also an issue if connection between SD reader and SDCard  is subject to EMI noise and so signal is disturbed which may loose data, and to analyze that it would need to check your complete house which won't fit by mail 
 
 I am sorry I cannot help more on your issue
